.github/workflows/conformance-report.yml

```diff
@@ -19,7 +19,11 @@ jobs:
         with:
           distribution: 'corretto'
           java-version: 17
-          cache: gradle
```

Comment:
Using the `actions/setup-java@v3` used to pull in Gradle 7.*. In some recent change to that action, Gradle 8.* was getting pulled in which caused build failures. Using Gradle's `gradle/gradle-build-action@v2` plugin allows us to fix the version to a specific version.